languages/javascript/scripts/scaffold/template/---.js

```diff
@@ -0,0 +1 @@
+/*---*/
```

Comment:
Can we instead have a quick comment on what this file should be, perhaps _also_ with a link to the "create a concept exercise" with the correct heading anchor?

Reply:
Same for _all_ the files

Reply:
Got it.